### DISCUSSION TITLE
What capital was once known as Batavia?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#574] [Galaxy Trivia] [answered]
**Hint:** until 1949

**Comment by peterbaccaro** [comment]
The capital that was once known as Batavia is **Jakarta**, the current capital of Indonesia. Batavia was the name given to the city by the Dutch during their colonial rule in Indonesia. The name Batavia was derived from the Latin name for the Netherlands, which is Batavus. The Dutch East India Company established Batavia in the early 17th century as the centre of their trading activities in the region. After Indonesia gained independence in 1945, the city's name was changed to Jakarta.

**Comment by SebastienXia** [comment, marked as answer]
Jakarta

**Comment by Veercodeprog** [comment]
The capital that was once known as Batavia is Jakarta, which is the current capital of Indonesia. Batavia was the name of the city during the Dutch colonial period in Indonesia, from the early 17th century until Indonesia's independence in 1945. After Indonesia's independence, the city's name was changed to Jakarta, which is now a megacity with a population of over 10 million people.

**Comment by alejandroch1202** [comment]
The capital that was once known as Batavia is Jakarta, the current capital of Indonesia. Batavia was the name given to the city by the Dutch during their colonial rule in Indonesia. The name Batavia was derived from the Latin name for the Netherlands, which is Batavus. The Dutch East India Company established Batavia in the early 17th century as the centre of their trading activities in the region. After Indonesia gained independence in 1945, the city's name was changed to Jakarta.
